settings.gradle

```diff
@@ -1,6 +1,7 @@
 pluginManagement {
 	repositories {
 		jcenter()
+		mavenCentral()
```

Comment:
for what? This is already covered by jcenter

Reply:
Probably leftover from some debugging I did earlier